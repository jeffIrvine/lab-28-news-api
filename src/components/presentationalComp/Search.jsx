import React from 'react';
import PropTypes from 'prop-types';

const Search = ({ handleClick, handleSubmit }) => (
  <div> 
    <form handleSubmit = { handleSubmit }>
      <input value="search" onChange={ handleClick }></input>
      <button> Search </button>
    </form>
  </div>
);

Search.propTypes = {
  handleClick: PropTypes.func.isRequired,
  handleSubmit: PropTypes.func.isRequired
};

export default Search;
