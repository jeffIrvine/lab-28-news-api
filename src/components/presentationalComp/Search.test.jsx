import React from 'react';
import Search from './Search';
import { render, cleanup } from '@testing-library/react';

describe('Search component', () => {
  afterEach(() => cleanup());
  it('renders Search', () => {
    const { asFragment } = render(<Search />);
    expect(asFragment()).toMatchSnapshot();
  });
});
