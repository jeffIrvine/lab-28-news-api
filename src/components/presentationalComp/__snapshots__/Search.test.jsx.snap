// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Search component renders Search 1`] = `
<DocumentFragment>
  <div>
    <input
      type="text"
      value=""
    />
  </div>
</DocumentFragment>
`;
